src/set_theory/game/pgame.lean

```diff
@@ -607,34 +607,44 @@ begin
   congr; funext i; cases i
 end
 
-/-- An explicit equivalence between the moves for Left in `-x` and the moves for Right in `x`. -/
--- This equivalence is useful to avoid having to use `cases` unnecessarily.
-def left_moves_neg (x : pgame) : (-x).left_moves ≃ x.right_moves :=
+@[simp] theorem left_moves_neg (x : pgame) : (-x).left_moves = x.right_moves :=
 by { cases x, refl }
 
-/-- An explicit equivalence between the moves for Right in `-x` and the moves for Left in `x`. -/
-def right_moves_neg (x : pgame) : (-x).right_moves ≃ x.left_moves :=
+@[simp] theorem right_moves_neg (x : pgame) : (-x).right_moves = x.left_moves :=
 by { cases x, refl }
 
-@[simp] lemma move_right_left_moves_neg {x : pgame} (i : left_moves (-x)) :
-  move_right x ((left_moves_neg x) i) = -(move_left (-x) i) :=
-begin
-  induction x,
-  exact (neg_neg _).symm
-end
-@[simp] lemma move_left_left_moves_neg_symm {x : pgame} (i : right_moves x) :
-  move_left (-x) ((left_moves_neg x).symm i) = -(move_right x i) :=
+/-- Turns a right move for `x` into a left move for `-x`. -/
+def to_left_moves_neg {x : pgame} (i : x.right_moves) : (-x).left_moves :=
+cast (left_moves_neg x).symm i
+
+/-- Turns a left move for `x` into a right move for `-x`. -/
+def to_right_moves_neg {x : pgame} (i : x.left_moves) : (-x).right_moves :=
+cast (right_moves_neg x).symm i
+
+/-- Turns a right move for `-x` into a left move for `x`. -/
+def neg_to_left_moves {x : pgame} (i : (-x).right_moves) : x.left_moves :=
+cast (right_moves_neg x) i
+
+/-- Turns a left move for `-x` into a right move for `x`. -/
+def neg_to_right_moves {x : pgame} (i : (-x).left_moves) : x.right_moves :=
+cast (left_moves_neg x) i
```

Comment:
Can't you provide the equivalences `cast (left_moves_neg x)` and `cast (right_moves_neg x)` instead? This will make goals of the form `⇑e x` where `e` is an `equiv`, and that has more simp lemmas.

Reply:
Wouldn't `e x` just evaluate to this same cast anyways?

Reply:
Yes, but yours gives `e x` for a random function `e` while mine gives `⇑e x` for `e` an `equiv`. They are defeq but not synctatically the same, and this is important here. Actually, I don't see how your current change is an improvement because you will rarely make use of the equality (because it's an equality of types), while the equivalence is crucial.

Reply:
I'd argue that once we have the basic API for turning moves of one type into moves of an equivalent type, neither the type equality nor equivalence are relevant anymore. 

Perhaps my previous PR #13611 illustrates this better. Once we're able to turn left/right moves of a sum into left or right moves of the addends or viceversa, everything works smoothly.

If it were really needed, it's trivial to convert the type equality into an equivalence (via `equiv.cast`) while the former can't be done directly.

Reply:
Still, why don't you promote your new functions to equivalences?

Reply:
Oh, I see what you mean now. I thought you were suggesting doing this:
```lean
def to_left_moves_neg {x : pgame} (i : x.right_moves) : (-x).left_moves :=
equiv.cast (left_moves_neg x).symm i
```
I'll implement this suggestion as soon as I implement it on the `pgame.add` PR.

Reply:
Yeah, the API became way nicer with these equivalences. Thanks!

Reply:
I'll implement a similar suggestion on the `pgame.add` PR.